www/docs/server/middlewares.md

```diff
@@ -111,11 +111,11 @@ export const appRouter = router({
 });
 ```
 
-## Context Swapping
+## Context Extension
 
-Context swapping in tRPC is a very powerful feature that allows you to create base procedures that dynamically infers new context in a flexible and typesafe manner.
+"Context Extension" in tRPC is a very powerful feature of middlewares, where you can dynamically add and override keys on a base procedure's context in a typesafe manner.
 
-Below we have an example of a middleware that changes properties of the context, and procedures will receive the new context value:
+Below we have an example of a middleware that changes properties of a context, the change(s) are then available to all chained consumers, such as other middlewares and procedures:
```

Comment:
```suggestion
Below we have an example of a middleware that changes properties of a context, the changes are then available to all chained consumers, such as other middlewares and procedures:
```

"properties" is already plural and I think it's clear enough that multiple changes and a singular change will function in the same way